File: Day1.java
import java.util.Scanner;

/**
 * Created by sophianguyen on 2016-05-02.
 */

public class Day1 {

    public static void main(String[] args) {
        int i = 4;
        double d = 4.0;
        String s = "HackerRank ";

        Scanner scan = new Scanner(System.in);

         /* Declare second integer, double, and String variables. */
        int a;
        double b;
        String c;
        String e;

        /* Read and save an integer, double, and String to your variables.*/

        System.out.println("Please enter an integer.");
        a = scan.nextInt();

        System.out.println("Now enter a double.");
        b = scan.nextDouble();

        System.out.println("Finally, enter a string!");
        c = scan.nextLine();
        e = scan.nextLine();

        /* Print the sum of both integer variables on a new line. */

        int sum = i + a;
        System.out.println(sum);

        /* Print the sum of the double variables on a new line. */

        double sum2 = d + b;
        System.out.println(sum2);

        /* Concatenate and print the String variables on a new line;
        	the 's' variable above should be printed first. */

        String concat = s.concat(e);
        System.out.println(concat);

        scan.close();

    }
}

File: Day1.cpp
//
//  Day1.cpp
//  HackerRank 30DaysOfCode
//
//  Created by Sophia Nguyen on 2016-05-03.
//  Copyright © 2016 Sophia Nguyen. All rights reserved.
//

#include <stdio.h>
#include <iostream>
#include <iomanip>
#include <limits>

using namespace std;

int main() {

int i = 4;
double d = 4.0;
string s = "HackerRank ";

// Declare second integer, double, and String variables.
int a;
double b;
string hi;

// Read and save an integer, double, and String to your variables.
cin >> a;
cin >> b;
cin.ignore(1, '\n');

getline(cin, hi);

// Print the sum of both integer variables on a new line.
cout << i + a << endl;

// Print the sum of the double variables on a new line.

cout << fixed << setprecision(1) << d + b << endl;

// Concatenate and print the String variables on a new line
// The 's' variable above should be printed first.

std::string concat = s + hi;
std::cout << concat << endl;


return 0;

}